src/cljs/main/broadfcui/page/workspace/method_configs/tab.cljs

```diff
@@ -37,26 +37,31 @@
           {:empty-message "There are no method configurations to display."
            :toolbar
            (add-right
-            [comps/Button {:text "Import Configuration..."
-                           :disabled? (case locked?
-                                        nil "Looking up workspace status..."
-                                        true "This workspace is locked."
-                                        false)
-                           :onClick #(modal/push-modal
-                                      [comps/OKCancelForm
-                                       {:header "Import Method Configuration"
-                                        :content
-                                        [:div {:style {:backgroundColor "white" :padding "1ex" :width 1000}}
-                                         [MethodConfigImporter {:workspace-id (:workspace-id props)
-                                                                :after-import (fn [{:keys [config-id]}]
-                                                                                (modal/pop-modal)
-                                                                                ((:on-config-imported props) config-id))}]]}])}])
+            [comps/Button
+             {:text "Import Configuration..."
+              :disabled? (case locked?
+                           nil "Looking up workspace status..."
+                           true "This workspace is locked."
+                           false)
+              :onClick #(modal/push-modal
+                         [comps/OKCancelForm
+                          {:header "Import Method Configuration"
+                           :content
+                           [:div {:style {:backgroundColor "white" :padding "1ex" :width 1000}}
```

Comment:
Nooooooo `ex` nooooooooo
And you can almost definitely get rid of `:width` here.